ui-v2/app/components/healthcheck-status.js

```diff
@@ -1,5 +1,11 @@
 import Component from '@ember/component';
-
+import { subscribe } from 'consul-ui/utils/computed/purify';
 export default Component.extend({
-  classNames: ['healthcheck-status'],
+  tagName: '',
+  count: subscribe('value', function(value) {
+    if (Array.isArray(value)) {
+      return value.length;
+    }
+    return value;
+  }),
 });
```

Comment:
Again `subscribe` is handy here, but not completely necessary. Potentially, we could hoist the function itself up and out of this file and reuse it here and above in `healthcheck-info`